docs/quickstart.md

```diff
@@ -0,0 +1,766 @@
+---
+title: QuickStart
+description: Powertools introduction
+---
+Quickstart introducing core Powertools functionalities.
+
+
+## Installation 
+With [SAM](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) installed, you can follow this Quickstart step by step or you can build a project with the final Quickstart code you can play with.
+If you follow it all along, create a new empty project.
+=== "shell"
+```bash
+sam init --runtime python3.9 --dependency-manager pip --app-template hello-world --name powertools-quickstart
```

Comment:
Great call out in using long parameter names to inform what each flag does!

Reply:
explicit is better than implicit :-) [ Zen of Python](https://towardsdatascience.com/the-zen-of-python-a-guide-to-pythons-design-principles-93f3f76d088a)